src/main/daml/Daml/Finance/Instrument/Generic/Util.daml

```diff
@@ -189,7 +189,7 @@ processClockUpdate actor eventCid _ self instrument observableCids enableNetting
   (remaining, pending) <- lifecycleClaims observableCids acquisitionTime claims [timeEvent v.eventTime]
 
   let
-    pendingAfterNetting = if enableNetting then net pending else pending
+    pendingAfterNetting = netOnTag pending
```

Comment:
I think we could even `netOnTag` by default on the base lifecycling utility. Not sure what you and @georg-da think about it

Reply:
Can we think of an example where claims must live in a single tag, but we don't want them to be netted? If not, then I agree we can generally do this.

Reply:
Mm, currently we have the generic instrument which is represented as a single tree. In that case you might want to avoid netting payments, but I think the solution is rather to have a generic instrument with a set of `TaggedClaims`.

Reply:
I was actually thinking about whether if was ok to remove the `enableNetting` flag at all (i.e. if we want `netOnTag` for all the different Bonds/Swaps), but I decided it was fine. Doing it in `processClockUpdate`, as in this PR, is maybe a reasonable compromise?

Reply:
I think it's a reasonable compromise